tests/e2e/basic/_basic_test_setup/setup_e2e_vlan.py

```diff
@@ -0,0 +1,181 @@
+import pytest
+
+
+@pytest.mark.sanity
+@pytest.mark.setup_vlan
+@pytest.mark.vlan
+@pytest.mark.parametrize(
+    'setup_profiles, create_profiles',
+    [(["VLAN"], ["VLAN"])],
+    indirect=True,
+    scope="class"
+)
+@pytest.mark.usefixtures("setup_profiles")
+@pytest.mark.usefixtures("create_profiles")
+class TestSetupvlan:
+
+    # @pytest.mark.open
```

Comment:
Remove code.

Reply:
that i commented to have to have a template that is required to add more ssid security modes in the configuration part